test/integration/plugins/storpool/test-storpool-tiers.py

```diff
@@ -0,0 +1,549 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@slavkap , file name convention dictates '_'s and not '-'s . can you rename this file?

Reply:
done